### PR TITLE
pillar description component

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

File: src/components/pillarDescription.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
```
Comment (asasmith) [inline src/components/pillarDescription.js:2]: If we're not using this now let's omit it

**Review by asasmith — CHANGES_REQUESTED** [review]

File: src/components/pillarDescription.js (hunk 1 above)
Comment (kcbreuil) [reply src/components/pillarDescription.js:2]: @asasmith We will need it once we refactor to export the data object

Comment (kcbreuil) [reply src/components/pillarDescription.js:2]: @asasmith I can update now if you'd like?

Comment (asasmith) [reply src/components/pillarDescription.js:2]: please. Let's avoid this pattern as much as possible.

**Review by asasmith — APPROVED** [review]

**Pull request merged**